core/bin/external_node/src/config/mod.rs

```diff
@@ -282,6 +285,10 @@ impl OptionalENConfig {
         10
     }
 
+    const fn default_enum_index_migration_chunks() -> u16 {
+        1
```

Comment:
The default value for EN is not yet calibrated and will be updated after tests on staging